src/CodeExamples.sln

```diff
@@ -23,6 +23,40 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "HashMap", "HashMap\HashMap.
+# EndProject
+# Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "HashMapNamespace.ProvideCapacity", "HashMap.Test\HashMapNamespace.ProvideCapacity\HashMapNamespace.ProvideCapacity.csproj", "{6B3046FE-22DE-43B3-9335-83883D159BA1}"
+# EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "HashMapNamespace.Add", "HashMap.Test\HashMapNamespace.Add\HashMapNamespace.Add.csproj", "{8513B597-4E9C-44CD-809F-290F446E21E4}"
+EndProject
+# Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "HashMapNamespace.AddOrUpdate", "HashMap.Test\HashMapNamespace.AddOrUpdate\HashMapNamespace.AddOrUpdate.csproj", "{DD451542-3BE6-47B2-8D29-B614A0EE2F6B}"
+# EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "HashMapNamespace.Remove", "HashMap.Test\HashMapNamespace.Remove\HashMapNamespace.Remove.csproj", "{2FB8A7E4-D031-466B-8834-DA4DA8265808}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "HashMapNamespace.ContainsKey", "HashMap.Test\HashMapNamespace.ContainsKey\HashMapNamespace.ContainsKey.csproj", "{49821606-F17E-4E4A-A1D0-5F5D052120F9}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "HashMapNamespace.TryGetValue", "HashMap.Test\HashMapNamespace.TryGetValue\HashMapNamespace.TryGetValue.csproj", "{E252454D-7F25-45DC-9CF1-C6DB354EB67D}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "HashMapNamespace.Clear", "HashMap.Test\HashMapNamespace.Clear\HashMapNamespace.Clear.csproj", "{C54A2E51-5AAA-42BE-BF3D-C5BF7FD9C557}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "HashMapNamespace.GetEnumerator", "HashMap.Test\HashMapNamespace.GetEnumerator\HashMapNamespace.GetEnumerator.csproj", "{CB575984-6982-4487-8944-B4751E0970F0}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "HashMapNamespace.HashMap64Node", "HashMap.Test\HashMapNamespace.HashMap64Node\HashMapNamespace.HashMap64Node.csproj", "{1ABBB916-2C9C-4558-95EE-EDDCEBB4A8A1}"
+EndProject
+# Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "HashMapNamespace.ToString", "HashMap.Test\HashMapNamespace.ToString\HashMapNamespace.ToString.csproj", "{393CAC49-92C5-44D5-9019-96179864A0FF}"
+# EndProject
+# Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "RealtimePlugin.Run", "RealtimePlugin.Test\RealtimePlugin.Run\RealtimePlugin.Run.csproj", "{75DCAF84-C05A-4CFD-A0EF-A03426D009D4}"
+# EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "SimpleClassLibrary.ComposePlugins", "SimpleClassLibrary.Test\SimpleClassLibrary.ComposePlugins\SimpleClassLibrary.ComposePlugins.csproj", "{EF2FF9B2-CE63-4261-B1B3-931D8491428E}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "SimpleClassLibrary.LoadAndRunPlugins", "SimpleClassLibrary.Test\SimpleClassLibrary.LoadAndRunPlugins\SimpleClassLibrary.LoadAndRunPlugins.csproj", "{0B70D31A-0736-434E-A7E3-3A460F050CB3}"
```

Comment:
### 5/5 Tests Pass.
```
vscode ➜ .../CodeExamples/src/SimpleClassLibrary.Test/SimpleClassLibrary.LoadAndRunPlugins (roost-1709036480) $ dotnet test
  Determining projects to restore...
  Restored /workspaces/CodeExamples/src/SimpleClassLibrary.Test/SimpleClassLibrary.LoadAndRunPlugins/SimpleClassLibrary.LoadAndRunPlugins.csproj (in 466 ms).
  2 of 3 projects are up-to-date for restore.
  PluginInterface -> /workspaces/CodeExamples/src/PluginInterface/bin/Debug/net6.0/PluginInterface.dll
  SimpleClassLibrary -> /workspaces/CodeExamples/src/SimpleClassLibrary/bin/Debug/net6.0/SimpleClassLibrary.dll
/workspaces/CodeExamples/src/SimpleClassLibrary.Test/SimpleClassLibrary.LoadAndRunPlugins/LoadAndRunPlugins.cs(117,58): warning CS8625: Cannot convert null literal to non-nullable reference type. [/workspaces/CodeExamples/src/SimpleClassLibrary.Test/SimpleClassLibrary.LoadAndRunPlugins/SimpleClassLibrary.LoadAndRunPlugins.csproj]
/workspaces/CodeExamples/src/SimpleClassLibrary.Test/SimpleClassLibrary.LoadAndRunPlugins/LoadAndRunPlugins.cs(110,13): warning NUnit2005: Consider using the constraint model, Assert.That(actual, Is.EqualTo(expected)), instead of the classic model, Assert.AreEqual(expected, actual) (https://github.com/nunit/nunit.analyzers/tree/master/documentation/NUnit2005.md) [/workspaces/CodeExamples/src/SimpleClassLibrary.Test/SimpleClassLibrary.LoadAndRunPlugins/SimpleClassLibrary.LoadAndRunPlugins.csproj]
/workspaces/CodeExamples/src/SimpleClassLibrary.Test/SimpleClassLibrary.LoadAndRunPlugins/LoadAndRunPlugins.cs(120,13): warning NUnit2005: Consider using the constraint model, Assert.That(actual, Is.EqualTo(expected)), instead of the classic model, Assert.AreEqual(expected, actual) (https://github.com/nunit/nunit.analyzers/tree/master/documentation/NUnit2005.md) [/workspaces/CodeExamples/src/SimpleClassLibrary.Test/SimpleClassLibrary.LoadAndRunPlugins/SimpleClassLibrary.LoadAndRunPlugins.csproj]
/workspaces/CodeExamples/src/SimpleClassLibrary.Test/SimpleClassLibrary.LoadAndRunPlugins/LoadAndRunPlugins.cs(134,13): warning NUnit2005: Consider using the constraint model, Assert.That(actual, Is.EqualTo(expected)), instead of the classic model, Assert.AreEqual(expected, actual) (https://github.com/nunit/nunit.analyzers/tree/master/documentation/NUnit2005.md) [/workspaces/CodeExamples/src/SimpleClassLibrary.Test/SimpleClassLibrary.LoadAndRunPlugins/SimpleClassLibrary.LoadAndRunPlugins.csproj]
/workspaces/CodeExamples/src/SimpleClassLibrary.Test/SimpleClassLibrary.LoadAndRunPlugins/LoadAndRunPlugins.cs(148,13): warning NUnit2005: Consider using the constraint model, Assert.That(actual, Is.EqualTo(expected)), instead of the classic model, Assert.AreEqual(expected, actual) (https://github.com/nunit/nunit.analyzers/tree/master/documentation/NUnit2005.md) [/workspaces/CodeExamples/src/SimpleClassLibrary.Test/SimpleClassLibrary.LoadAndRunPlugins/SimpleClassLibrary.LoadAndRunPlugins.csproj]
/workspaces/CodeExamples/src/SimpleClassLibrary.Test/SimpleClassLibrary.LoadAndRunPlugins/LoadAndRunPlugins.cs(169,13): warning NUnit2005: Consider using the constraint model, Assert.That(actual, Is.EqualTo(expected)), instead of the classic model, Assert.AreEqual(expected, actual) (https://github.com/nunit/nunit.analyzers/tree/master/documentation/NUnit2005.md) [/workspaces/CodeExamples/src/SimpleClassLibrary.Test/SimpleClassLibrary.LoadAndRunPlugins/SimpleClassLibrary.LoadAndRunPlugins.csproj]
  SimpleClassLibrary.LoadAndRunPlugins -> /workspaces/CodeExamples/src/SimpleClassLibrary.Test/SimpleClassLibrary.LoadAndRunPlugins/bin/Debug/net7.0/SimpleClassLibrary.LoadAndRunPlugins.dll
Test run for /workspaces/CodeExamples/src/SimpleClassLibrary.Test/SimpleClassLibrary.LoadAndRunPlugins/bin/Debug/net7.0/SimpleClassLibrary.LoadAndRunPlugins.dll (.NETCoreApp,Version=v7.0)
Microsoft (R) Test Execution Command Line Tool Version 17.7.2 (x64)
Copyright (c) Microsoft Corporation.  All rights reserved.

Starting test execution, please wait...
A total of 1 test files matched the specified pattern.
No plugins with type: validType

Loaded plugin: 
Plugin returned NoError


Passed!  - Failed:     0, Passed:     5, Skipped:     0, Total:     5, Duration: 177 ms - SimpleClassLibrary.LoadAndRunPlugins.dll (net7.0)
vscode ➜ .../CodeExamples/src/SimpleClassLibrary.Test/SimpleClassLibrary.LoadAndRunPlugins (roost-1709036480) $ 
```